pkgs/by-name/zf/zfs-replicate/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  python3Packages,
+  fetchFromGitHub,
+  lz4,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zfs_replicate";
+  version = "4.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "alunduil";
+    repo = "zfs-replicate";
+    tag = "v${version}";
+    hash = "sha256-VajMSoFZ4SQXpuF1Lo6S9IhxvspCfUwpNw5zg16uA3M=";
+  };
+
+  # For compression to work, both local and remote systems must have lz4 installed.
+  # This hard codes the path to the lz4 when running it on the local system.
+  postPatch = ''
+    substituteInPlace zfs/replicate/compress/command.py \
+      --replace-fail '("/usr/bin/env - lz4' '("${lib.getExe lz4}'
```

Comment:
Did you confirm this only changes the local invocation? I can't recall but could see this being the same string for both local and remote.

Reply:
Also, shouldn't this be lz4c now?